tests/plugins/workday.py

```diff
@@ -23,6 +23,9 @@
 
 from __future__ import annotations
 
-from airflow.example_dags.plugins.workday import AfterWorkdayTimetable, WorkdayTimetablePlugin
+from airflow.example_dags.plugins.workday import (
+    AfterWorkdayTimetable as TestAfterWorkdayTimetable,
+    WorkdayTimetablePlugin as TestWorkdayTimetablePlugin,
+)
```

Comment:
Why are these needed?

Reply:
Okay, in the attempt to fix all problems with pytests I reverted this and moved the examples and the test base all to example DAGs, this is no needed anymore.